Gemfile

```diff
@@ -27,7 +27,8 @@ gem 'puma'
 
 gem 'bootsnap', require: false
 
-gem 'exception_notification'
+gem 'exception_notification', git: 'https://github.com/smartinez87/exception_notification.git',
```

Comment:
Will changes to this gem still come through depfu's? I imagine since its off a git branch updates will be automatic as to when this branch is pushed to?